packages/image/README.md

```diff
@@ -0,0 +1,82 @@
+# @rollup/plugin-image
+
+🍣 A Rollup plugin which imports JPG, PNG, GIF, SVG, and WebP files.
+
+Images are encoded using base64, which means they will be 33% larger than the size on disk. You should therefore only use this for small images where the convenience of having them available on startup (e.g. rendering immediately to a canvas without co-ordinating asynchronous loading of several images) outweighs the cost.
+
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v8.0.0+) and Rollup v1.20.0+.
+
+## Install
+
+Using npm:
+
+```console
+npm install @rollup/plugin-dsv --save-dev
+```
+
+## Usage
+
+Assuming a `src/index.js` exists and contains code like the following:
+
+```js
+import logo from "./rollup.png";
+
+console.log(logo);
+```
+
+Create a `rollup.config.js` [configuration file](https://www.rollupjs.org/guide/en/#configuration-files) and import the plugin:
+
+```js
+import image from "@rollup/plugin-image";
+
+export default {
+  input: "src/index.js",
```

Comment:
Maybe use single quotes here to match the package code and other READMEs

Reply:
Nevermind, looks like other packages are switching

Reply:
No that's actually an error with the MD linting from prettier. I'll look into it.